package.json

```diff
@@ -97,8 +97,8 @@
     "build": "DISABLE_ESLINT_PLUGIN=true node scripts/build.js",
     "test": "node scripts/test.js --transformIgnorePatterns \"node_modules/(?!@apollo/client/link/core/ApolloLink.js)/\"",
     "test:nowatch": "node scripts/test.js --watchAll=false  --transformIgnorePatterns \"node_modules/(?!@apollo/client/link/core/ApolloLink.js)/\"",
-    "lint-check": "tsc --noEmit && eslint src/**/*.ts{,x}",
-    "lint-fix": "tsc --noEmit && eslint src/**/*.ts{,x} --fix",
+    "lint-check": "eslint src/**/*.ts{,x}",
+    "lint-fix": "eslint src/**/*.ts{,x} --fix",
```

Comment:
This would have been set up by us manually back in the day AFAIK, perhaps with the ejection from `react-scripts` and major version update to ESLint this is no longer needed?